mitiq/mitiq_qiskit/conversions.py

```diff
@@ -128,6 +128,33 @@ def _map_bits(
     return [Bit(new_registers[i], j) for i, j in mapped_indices]
 
 
+def _measurement_order(circuit: qiskit.QuantumCircuit):
+    """Returns the left-to-right measurement order in the circuit.
+
+    The "measurement order" is a list of tuples (qubit, bit) involved in
+    measurements ordered as they appear going left-to-right through the circuit
+    (i.e., iterating through circuit.data). The purpose of this is to be able
+    to do
+
+    >>> for (qubit, bit) in _measurement_order(circuit):
+    >>>     other_circuit.measure(qubit, bit)
+
+    which ensures ``other_circuit`` has the same measurement order as
+    ``circuit``, assuming ``other_circuit`` has the same register(s) as
+    ``circuit``.
+
+    Args:
+        circuit: Qiskit circuit to get the measurement order of.
+    """
+    order = []
+    for (gate, qubits, cbits) in circuit.data:
+        if isinstance(gate, qiskit.circuit.Measure):
+            # TODO: Is it guaranteed that a measurement gate only has one qubit
+            #  and one cbit in Qiskit?
```

Comment:
One can define measurements with more qubits/cbits. However I think they are mapped always to single-qubit-cbit measurements. E.g. 
```
>>> q = QuantumRegister(2)
>>> c = ClassicalRegister(2)
>>> qc = QuantumCircuit(q, c)
>>> qc.measure(q, c)
<qiskit.circuit.instructionset.InstructionSet object at 0x7fd7d70a0f50>
>>> qc.draw()
         ┌─┐   
q0_0: |0>┤M├───
         └╥┘┌─┐
q0_1: |0>─╫─┤M├
          ║ └╥┘
 c0_0: 0 ═╩══╬═
             ║ 
 c0_1: 0 ════╩═
 
>>> qc.data
[(<qiskit.circuit.measure.Measure object at 0x7fd7d708d7d0>, [Qubit(QuantumRegister(2, 'q0'), 0)], [Clbit(ClassicalRegister(2, 'c0'), 0)]), (<qiskit.circuit.measure.Measure object at 0x7fd7d708d7d0>, [Qubit(QuantumRegister(2, 'q0'), 1)], [Clbit(ClassicalRegister(2, 'c0'), 1)])]

```
Given this fact, what about if we replace the TODO with:
`assert len(qubits) == len(cbits) == 1`

Reply:
> However I think they are mapped always to single-qubit-cbit measurements.

Ok, this is what I thought as well. I updated the todo with an exception and "please let us know" if there is ever more than one qubit/cbit in the measurement.